.gitignore

```diff
@@ -0,0 +1,56 @@
+# sed-tools
```

Comment:
I think this file should be added in on a  separate commit

Reply:
Agreed! Will separate.